podcasts/EpisodeDetailViewController+Actions.swift

```diff
@@ -51,10 +51,20 @@ extension EpisodeDetailViewController {
             if !PlaybackManager.shared.playing() {
                 dismiss(animated: true, completion: nil)
             }
+            if let timestamp = timestamp {
+                DataManager.sharedManager.saveEpisode(playedUpTo: timestamp, episode: episode, updateSyncFlag: true)
```

Comment:
Why do we need `updateSyncFlag` to be `true`? Can't we just let that be for the app to handle when the user is playing the episode?

Reply:
I will follow your suggestion and test it again to see if all is good.